tests/unit/h/views/accounts_test.py

```diff
@@ -681,7 +681,7 @@ def test_post_email_form_with_valid_data_changes_email(
         self, form_validating_to, pyramid_request
     ):
         controller = views.AccountController(pyramid_request)
-        controller.forms["email"] = form_validating_to({"email": "new_email_address"})
+        controller.email_form = form_validating_to({"email": "new_email_address"})
```

Comment:
The existing tests keep writing `controller.forms` which is terrible test writing. For now just updating them to fit with the refactored `AccountController` which no longer uses a `self.forms` dict.